spec/factories/scope_limitations.rb

```diff
@@ -0,0 +1,17 @@
+FactoryBot.define do
+  factory :scope_limitation do
+    sequence(:code) { |n| format('AA%03d', n) }
+    meaning { 'Meaning' }
+    description { 'Description' }
+    substantive { true }
+    delegated_functions { false }
+  end
+
+  trait :with_real_data do
```

Comment:
I think the `trait` should be defined inside the `factory :scope_limitation` block. Like for the other factories.